cohort_3/curriculum/workshops/maestro-website/README.md

```diff
@@ -0,0 +1,110 @@
+# Maestro + webpage workshop
+
+## Website bootstrap
+
+- Open a cloud9 workspace and create a new folder titled `texting-site`
+- Inside the folder create a new file called `index.html`
+- Make a basic website template with a `<head>` and `<body>`
+
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+
+  </head>
+  <body>
+
+  </body>
+</html>
+```
```

Comment:
Can you indent this to be with the bullet point?

Ex.

```
- This is a bullet point

  ```
  some code for the bullet point
  ```
```